mne/surface.py

```diff
@@ -423,11 +439,18 @@ def read_surface(fname, verbose=None):
     tris : int array, shape=(n_faces, 3)
         Triangulation (each line contains indexes for three points which
         together form a face).
-
+    volume_info : dict-like
+        If read_metadata is true, key-value pairs found in the geometry file.
     See Also
     --------
     write_surface
     """
+    # XXX: Tests fail here due to numerical error.
```

Comment:
Can you link to it? I'm surprised, and we should probably sort it out. Although see comment below, you might need to cast as 
```
coords = coords.astype(np.float)  # XXX: due to mayavi bug on mac 32bits
```